scripts/prepare-release.sh

```diff
@@ -1,45 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Um, no? we use these

Reply:
You put them in IOGX? I think that's a bad idea. These are some things I hacked together for `plutus`: there is 0 guarantee they will work for anyone else, and I don't want to go through `iogx` if I want to hack on them further.